tests/unit/createBirthdaySquad.test.ts

```diff
@@ -0,0 +1,254 @@
+      getSquadMembers,
+      "getRandomSquadMembers",
+    );
+
+    openConversationMock.mockResolvedValueOnce(constants.converastionId);
+
+    await sendMockSqsMessage("createBirthdaySquad", event, handler);
+
+    expect(getRandomSquadMembersSpy).toHaveBeenCalledWith({
+      teamId: constants.teamId,
+      usersToExclude: [constants.birthdayPerson],
+      limit: BIRTHDAY_SQUAD_SIZE,
+    });
+
+    expect(openConversation).not.toBeCalled();
+  });
+
+  it("Should not call publishEvent", async () => {
+    const event = {
+      birthdayPerson: constants.birthdayPerson,
+      eventId: constants.eventId,
+      team: constants.teamId,
+    } satisfies Events["createBirthdaySquad"];
+    const getRandomSquadMembersSpy = vi.spyOn(
+      getSquadMembers,
+      "getRandomSquadMembers",
+    );
+
+    openConversationMock.mockResolvedValueOnce(constants.converastionId);
+
+    await sendMockSqsMessage("createBirthdaySquad", event, handler);
+
+    expect(getRandomSquadMembersSpy).toHaveBeenCalledWith({
+      teamId: constants.teamId,
+      usersToExclude: [constants.birthdayPerson],
+      limit: BIRTHDAY_SQUAD_SIZE,
+    });
+
+    expect(PutEventsCommand).not.toBeCalled();
+    expect(eventBridge.send).not.toBeCalled();
+  });
+});
+
+describe("Applied squad members is less the 3", () => {
+  let openConversationMock: Mock;
+
+  beforeEach(() => {
+    openConversationMock = openConversation as Mock;
+  });
+
+  afterEach(async () => {
+    vi.clearAllMocks();
+  });
+
+  it("Should call getRandomSquadMembers", async () => {
+    const event = {
+      birthdayPerson: constants.birthdayPerson,
+      eventId: constants.eventId,
+      team: constants.teamId,
+    } satisfies Events["createBirthdaySquad"];
+    const getRandomSquadMembersSpy = vi.spyOn(
+      getSquadMembers,
+      "getRandomSquadMembers",
+    );
+
+    openConversationMock.mockResolvedValueOnce(constants.converastionId);
+
+    await sendMockSqsMessage("createBirthdaySquad", event, handler);
+
+    expect(getRandomSquadMembersSpy).toHaveBeenCalledWith({
+      teamId: constants.teamId,
+      usersToExclude: [constants.birthdayPerson],
+      limit: BIRTHDAY_SQUAD_SIZE,
+    });
+  });
+});
+
+describe("Applied squad members is 3 or more", () => {
+  let openConversationMock: Mock;
+  let insertedSquadMembers: string[];
+
+  beforeEach(async () => {
+    openConversationMock = openConversation as Mock;
+
+    await testDb.insert(users).values({
+      id: constants.birthdayPerson,
+      teamId: constants.teamId,
+      birthday: dayjs.utc().toDate(),
+    });
+
+    await testDb.insert(users).values(
+      constants.otherUserIds.map((userId) => ({
+        id: userId,
+        teamId: constants.teamId,
+        birthday: dayjs.utc().toDate(),
+      })),
+    );
+
+    insertedSquadMembers = await seedSquadJoins(
+      constants.birthdayPerson,
+      constants.teamId,
+      constants.otherUserIds,
+      3,
+    );
+  });
+
+  afterEach(async () => {
```

Comment:
Shouldn't we also have a beforeAll clean? If another test doesn't clean up properly or something then we might not start with an empty db.